compiler/noirc_frontend/src/monomorphization/ast.rs

```diff
@@ -65,7 +67,20 @@ pub struct LocalId(pub u32);
 
 /// A function ID corresponds directly to an index of `Program::functions`
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
-pub struct FuncId(pub u32);
+// pub struct FuncId(pub u32);
+pub enum FuncId {
+    Dummy,
+    Interned(u32),
+}
```

Comment:
It looks like `FuncId::Dummy` is unused after the removal in take_main_body?

Reply:
Removed :+1: